### PR TITLE
Update user baselines

### PR DESCRIPTION
**Pull request by rbuckton**
Accepts the new user baselines from #42431

### CODE REVIEW

File: tests/baselines/reference/docker/xterm.js.log
Comment (sandersn) [inline tests/baselines/reference/docker/xterm.js.log:10]: these errors seem worse than before

File: tests/baselines/reference/docker/vscode.log
Comment (sandersn) [inline tests/baselines/reference/docker/vscode.log:1]: this is worse than before, right? Is this due to our test harness changing or vscode changing?

File: tests/baselines/reference/docker/azure-sdk.log
Comment (sandersn) [inline tests/baselines/reference/docker/azure-sdk.log:67]: looks like azure-sdk is still blocked by not installing (or a bad install?) `@typescript-eslint/scope-manager`

**Review by sandersn — COMMENTED** [review]
Looks like there's still work to do for xterm.js, vscode and azure-sdk. office-ui-fabric still looks super broken, but now in a different way. I guess it's fine to merge once #42431 is in, but it's still the case that the docker tests are half-useful at best.

File: tests/baselines/reference/docker/xterm.js.log
Comment (rbuckton) [reply tests/baselines/reference/docker/xterm.js.log:10]: I'm not certain we can know what broke this. None of the projects fixed by #42431 have been building or contributing updated logs for some time now, so there's no way to know *what* caused these breaks. All of these breaks are based on what is in master *now*.

File: tests/baselines/reference/docker/vscode.log
Comment (rbuckton) [reply tests/baselines/reference/docker/vscode.log:1]: We haven't been building `vscode` for awhile, so there's now way to know. This is what `vscode` reports for `master` *now* (after getting the build to actually run in #42431).

File: tests/baselines/reference/docker/azure-sdk.log
Comment (rbuckton) [reply tests/baselines/reference/docker/azure-sdk.log:67]: Is this our issue or an issue in `azure-sdk`? I didn't do anything for `azure-sdk` in #42431, so as far as I can tell this is just the latest build output from `master`.

File: tests/baselines/reference/docker/xterm.js.log
Comment (rbuckton) [reply tests/baselines/reference/docker/xterm.js.log:10]: The only thing I changed for xterm was the NodeJS version (since xterm started requiring features not present in NodeJS 8). This seems like a problem with their build, not our test (and these user tests often seem to run afoul of that).

**Comment by rbuckton** [comment]
This is purely to get us caught up now that we're actually running projects like `vscode`, `xterm.js`, `prettier`, `office-ui-fabric`, etc. that haven't been building (or contributing logs) for the past few weeks (at a minimum). If we want to run down build failures in the logs now that they are actually up to date with what's in `master`, I'd rather do that in a separate PR. 

The important thing is re-establishing our baseline for anything else we want to test for 4.2, since now that #42431 has merged everyone running the user tests will be seeing these due to an out-of-date baseline.

File: tests/baselines/reference/docker/vscode.log
Comment (rbuckton) [reply tests/baselines/reference/docker/vscode.log:1]: This is probably something we can ignore? The error reported is that `monaco.d.ts` changed (from [here](https://github.com/microsoft/vscode/blob/a095d7fcf74d0b072886c3041bd5932f3b5be445/build/lib/compilation.js#L171)), which is likely due to Anders's type alias naming PR. The error will likely go away once vscode is building using 4.2 and generates a stable copy of monaco.d.ts.

**Comment by rbuckton** [comment]
I need to merge this soon so I can move forward with running the user tests against several community PRs. Given this is a baseline update based on what's already in master (and not a functionality update), unless someone has a pressing concern, I'll probably merge this in the next 30 minutes.

**Review by Stirfry70 — APPROVED** [review]

**Pull request merged**
